common4j/src/main/com/microsoft/identity/common/java/flighting/CommonFlight.java

```diff
@@ -175,13 +175,18 @@ public enum CommonFlight implements IFlightConfig {
     /**
      * Flight to skip ests telemetry.
      */
-    SKIP_ESTS_TELEMETRY("SkipEstsTelemetry", false),
+    SKIP_ESTS_TELEMETRY("SkipEstsTelemetry", true),
```

Comment:
This has been true in Broker for more than a month now. Make it true by default here as well.